src/cli/commands/lint.ts

```diff
@@ -84,7 +85,7 @@ async function lint(name: string, flags: any, command: Lint) {
 
   let results = [];
   try {
-    results = await spectral.run(obj.data);
+    results = await spectral.run(obj.data, { uri: join(process.cwd(), name), ...obj });
```

Comment:
Can we avoid join cwd and use `path.resolve()` instead? We had this issue in prism where it would append onto _anything_ you passed in, which broke URLs, absolute paths, etc.